src/Lockedfield.php

```diff
@@ -165,7 +175,21 @@ public function getLocks($itemtype, $items_id)
 
         $locks = [];
         foreach ($iterator as $row) {
-            $locks[] = $row['field'];
+            if ($fields_only === true) {
+                $locks[] = $row['field'];
+            } else {
+                $value = $row['value'];
+                //if it refers to a ForeignKey get dropdown name
+                if (
+                    $row['field'] !== 'entities_id'
+                    && isForeignKeyField($row['field'])
+                    && is_a($itemtype = getItemtypeForForeignKeyField($row['field']), CommonDropdown::class, true)
+                    && null !== $row['value']
```

Comment:
1. From performance point of view, it will probably be more efficient to check null value at first.
2. I do not think that `entities_id` may be locked, but, in this case, we would probably want to display its name too.

```suggestion
                    $row['value'] !== null
                    && isForeignKeyField($row['field'])
                    && is_a($itemtype = getItemtypeForForeignKeyField($row['field']), CommonDropdown::class, true)
```